lib/netsuite/records/expense_category.rb

```diff
@@ -0,0 +1,20 @@
+module NetSuite
+  module Records
+    class ExpenseCategory
+      include Support::Fields
+      include Support::Actions
+      include Namespaces::ListAcct
+
+      actions :search
```

Comment:
We only know the search works at this point... I guess fine to add the others, we just haven't made sure they worked.